README.md

```diff
@@ -47,6 +47,37 @@ After that, you may run the binary as any normal program:
 ```
 ./shark-clipper-linux-x64-latest.bin
 ```
+#### Using the Mac Prebuilt Binary
```

Comment:
Missing a space after the section header.
Not all markdown interpreters want will allow a missing space here.